### PR TITLE
Setup cron and regular data dumps for hetzner

### PR DESCRIPTION
**Pull request by paramsingh**
### Problem

The cron and data dumps setup hadn't been migrated.

### Solution

* Add scripts to create dumps and copy to MusicBrainz FTP (ftp://ftp.musicbrainz.org/pub/musicbrainz/bookbrainz)
* Setup dockerfile with changes for the new cron service

### Areas of Impact

Not much really, but the bookbrainz.org/dumps/latest.bz2 link will need to redirect to ftp now.

### CODE REVIEW

**Comment by paramsingh** [comment]
I have this running on prince rn, so a dump should appear by tomorrow.

File: scripts/create-dumps.sh [hunk 1]
```diff
@@ -1,41 +1,46 @@
...
 # Switch directory
-pushd /home/bookbrainz/bookbrainz.org/bookbrainz-docker-prod/dumps
+pushd /home/bookbrainz/data/dumps
+
 
 DUMP_FILE=bookbrainz-dump-`date -I`.sql
 
-export PGPASSWORD="postgres"
+echo "Creating data dump..."
+export PGPASSWORD="bookbrainz"
```
Comment (MonkeyDo) [inline scripts/create-dumps.sh]: I was hoping we'd get rid of a hardcoded password entirely…
Should this be set as an env variable instead?

File: docker/prod/rc.local [hunk 2]
```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```
Comment (mayhem) [inline docker/prod/rc.local]: Isn't a beta version of this file missing?

File: docker/prod/rc.local [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+# BookBrainz rc.local
+# Enables a specific runit service depending on what the
+# CONTAINER_NAME variable is set to
+
+
+if [ "${CONTAINER_NAME}" = "bookbrainz-web-prod" ]
+then
+    rm -f /etc/service/webserver/down
```
Comment (mayhem) [inline docker/prod/rc.local]: This still looks wrong to me at first glance, but is really quite cool.

File: scripts/create-dumps.sh (hunk 1 above)
Comment (mayhem) [reply scripts/create-dumps.sh]: Our other databases do not have passwords -- we should probably work to get rid of this one too.

File: scripts/create-dumps.sh [hunk 4]
```diff
@@ -1,41 +1,46 @@
 #!/usr/bin/env bash
```
Comment (mayhem) [inline scripts/create-dumps.sh]: Let's start following https://github.com/metabrainz/metabrainz-howto-guides/blob/master/writing_effective_data_processing_scripts.md and have our data scripts return proper exit codes.

**Comment by paramsingh** [comment]
Ready for review again.

File: scripts/create-dumps.sh (hunk 4 above)
Comment (MonkeyDo) [reply scripts/create-dumps.sh]: @paramsingh 
Rob says: 
> a last line of `exit 0` should suffice.

Comment (paramsingh) [reply scripts/create-dumps.sh]: Done!

**Pull request merged**